Steps4Impact/Settings/TeamSettingsViewController.swift

```diff
@@ -71,11 +71,34 @@ extension TeamSettingsViewController: SettingsActionCellDelegate {
           viewController: self,
           shareButton: button,
           string: Strings.Share.item)
+    case .editname:
+      AKFCausesService.getParticipant(fbid: Facebook.id) { (result) in
+        guard let participant = Participant(json: result.response) else { return }
+        guard participant.team != nil else { return }
+        let alert = UIAlertController(title: Strings.TeamSettings.editTeamName,
+                                      message: Strings.TeamSettings.editTeamNameMessage,
+                                      preferredStyle: UIAlertController.Style.alert)
+        let action = UIAlertAction(title: Strings.TeamSettings.update, style: .default) { (_) in
+          let textField = alert.textFields![0] as UITextField
+          let newName = textField.text!
+          AKFCausesService.editTeamName(team: (participant.team?.id)!, name: newName) { (_) in
+            onMain {
+              alert.dismiss(animated: true, completion: nil)
+              self.navigationController?.popViewController(animated: true)
+              NotificationCenter.default.post(name: .teamChanged, object: nil)
+            }
+          }
+        }
+        alert.addTextField { (textField) in
+          textField.text = "Enter Team Name"
+        }
+        alert.addAction(action)
+        UIApplication.shared.keyWindow?.rootViewController?.present(alert, animated: true, completion: nil)
```

Comment:
Why is this being presented from the root view controller rather than this view controller?